README.md

```diff
@@ -248,6 +248,92 @@ The generated css will be:
   }
 ```
 
+## Overriding styles
+
+When combining multiple aphrodite styles, you **must** merge all of your styles into a single call to `css()`, and **must not** combine the generated class names that aphrodite outputs (via string concatenation, `classnames`, etc.).
```

Comment:
"must" has a specific meaning in RFC style, but I agree that it might read as a little forceful here.